### ISSUE TITLE
Optifine 1.15.2 problem.

### ISSUE THREAD

**Issue by MariusR32** [sp614x/optifine#3985]
I have some problems with optifine 1.15.2. When i run minecraft without optifine my fps are beetwen 90 and 150 and are preety stable but after i install optifine the fps are going all the way up to 300 fps but are not stable and sometimes i have bad spikes where the fps drops to 40 or less. I allocate 6Gb of ram to minecraft and i have smooth fps and smooth world enabled. Can somebody help me pls ??

**Comment by realjackasterisk** [comment]
More RAM =/= better. More RAM will in fact cause more instability due to how Java itself works. You should only allocate enough to the point where memory usage tops out at 75%. In most vanilla/OptiFine standalone installations, this will usually be 1-2 GB.
![image](https://user-images.githubusercontent.com/31686191/82715942-0c886600-9c4a-11ea-9a7c-0703aa787598.png)
The highlighted number will slowly rise over time. Every once in a while (maybe 10-20 seconds) it'll reset, going back to a lower number. As I said, this should only ever go up to ~75%. Simply put, with more memory allocated to the game, this will cause larger spikes in frame time when the number resets.

There's a more technical explanation on what's actually happening here, but... me, not being a Java developer, I'm definitely not the best person to explain it.

**Comment by realjackasterisk** [comment]
TL;DR Allocate *less* ram, unless you're also playing with other mods. Let me know if this works.

**Comment by MariusR32** [comment]
Yep, it works. Thanks a lot!!

sâm., 23 mai 2020, 02:52 Kyle Edwards <notifications@github.com> a scris:

**Comment by realjackasterisk** [comment]
👍 closing as solved, then
